File: python/sprint1_nonfinals/k.py
from typing import List, Tuple

def get_sum(number_list: List[int], k: int) -> List[int]:
    # Здесь реализация вашего решения
    pass

def read_input() -> Tuple[List[int], int]:
    n = int(input())
    number_list = list(map(int, input().strip().split()))
    k = int(input())
    return number_list, k

number_list, k = read_input()
print(" ".join(map(str, get_sum(number_list, k))))

File: csharp/sprint1_nonfinals/K.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Linq;

public class K
{
    private static TextReader reader;
    private static TextWriter writer;

    private static List<int> GetSum(List<int> numberList, int k)
    {
        // Ваше решение
    }

    public static void Main(string[] args)
    {
        reader = new StreamReader(Console.OpenStandardInput());
        writer = new StreamWriter(Console.OpenStandardOutput());

        var numberLength = ReadInt();
        var numberList = ReadList();
        var k = ReadInt();
        var sum = GetSum(numberList, k);

        writer.WriteLine(string.Join(" ", sum));

        reader.Close();
        writer.Close();
    }

    private static List<int> ReadList()
    {
        return reader.ReadLine()
            .Split(new[] { ' ', '\t', }, StringSplitOptions.RemoveEmptyEntries)
            .Select(int.Parse)
            .ToList();
    }

    private static int ReadInt()
    {
        return int.Parse(reader.ReadLine());
    }
}
